python/taichi/examples/graph/texture_graph.py

```diff
@@ -79,7 +79,7 @@ def main():
         mod.save(tmpdir, '')
     else:
         t = 0.0
-        window = ti.ui.Window('UV', res)
+        window = ti.ui.Window('UV', res, True)
```

Comment:
nit: is this `True` somehow required? If so it's better to write `vsync=True` ;)